<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: qradialgradient(spread:pad, cx:0.336, cy:0.471591, radius:0.911, fx:0.35, fy:0.5, stop:0 rgba(255, 255, 255, 255), stop:0.681159 rgba(0, 0, 0, 255));
</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="close">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>500</y>
      <width>121</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius:10px;
color:white;
background-color:gray;
</string>
    </property>
    <property name="text">
     <string>EXIT</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_photo">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>390</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:qlineargradient(spread:pad, x1:0.502507, y1:1, x2:0.503, y2:0.0170455, stop:0 rgba(119, 0, 219, 255), stop:1 rgba(234, 181, 255, 255));
border-radius:10px;</string>
    </property>
    <property name="text">
     <string>PHOTO</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_shutter">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>330</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:qlineargradient(spread:pad, x1:0.502507, y1:1, x2:0.503, y2:0.0170455, stop:0 rgba(247, 255, 0, 255), stop:1 rgba(255, 255, 255, 255));
border-radius:10px;
</string>
    </property>
    <property name="text">
     <string>SHUTTER</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_play">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>200</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:qlineargradient(spread:pad, x1:0.502507, y1:1, x2:0.503, y2:0.0170455, stop:0 rgba(0, 33, 255, 255), stop:1 rgba(164, 164, 214, 255));
border-radius:10px;



border-radius:10px;</string>
    </property>
    <property name="text">
     <string>PLAY</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_rec_stop">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>140</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:qlineargradient(spread:pad, x1:0.502507, y1:1, x2:0.503, y2:0.0170455, stop:0 rgba(255, 0, 0, 255), stop:1 rgba(214, 214, 214, 255));
border-radius:10px;</string>
    </property>
    <property name="text">
     <string>REC</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_video_Switch">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>520</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:qlineargradient(spread:pad, x1:0.502507, y1:1, x2:0.503, y2:0.0170455, stop:0 rgba(27, 121, 177, 255), stop:1 rgba(234, 181, 255, 255));
border-radius:10px;</string>
    </property>
    <property name="text">
     <string>VIDEO1</string>
    </property>
   </widget>
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>611</width>
      <height>461</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>3</number>
    </property>
    <widget class="QWidget" name="page">
     <widget class="QScrollArea" name="scrollArea">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>611</width>
        <height>461</height>
       </rect>
      </property>
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="scrollAreaWidgetContents">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>609</width>
         <height>459</height>
        </rect>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="page_3">
     <widget class="QScrollArea" name="scrollArea_2">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>609</width>
        <height>459</height>
       </rect>
      </property>
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="scrollAreaWidgetContents_2">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>607</width>
         <height>457</height>
        </rect>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="page_2">
     <widget class="QLabel" name="DisplayCapturedImage">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>611</width>
        <height>461</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_4">
     <widget class="QScrollArea" name="scrollArea_3">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>611</width>
        <height>461</height>
       </rect>
      </property>
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="scrollAreaWidgetContents_3">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>609</width>
         <height>459</height>
        </rect>
       </property>
      </widget>
     </widget>
    </widget>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>141</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(0, 0, 0, 0%);
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <zorder>pushButton_photo</zorder>
   <zorder>close</zorder>
   <zorder>pushButton_shutter</zorder>
   <zorder>pushButton_play</zorder>
   <zorder>pushButton_rec_stop</zorder>
   <zorder>pushButton_video_Switch</zorder>
   <zorder>stackedWidget</zorder>
   <zorder>label</zorder>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
